.travis.yml

```diff
@@ -10,15 +10,13 @@ php:
   - 7.0
   - 7.1
 
-  matrix:
+matrix:
   fast_finish: true
   include:
```

Comment:
@railto on doctrine/doctrine2#6288 I've managed to get the test suite running (but HHVM fails badly in terms of PHP compatibility that I gave up 😭).

What I've learned:

-  `dist` should be defined on the root level
- we must set `hhvm.php7.all=1` AND `hhvm.php7.scalar_types=0` otherwise it doesn't work properly (HHVM simply enables strict mode for all files)